pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -561,6 +561,11 @@ func (d *DependenciesDistributor) createOrUpdateAttachedBinding(attachedBinding
 	bindingKey := client.ObjectKeyFromObject(attachedBinding)
 	err := d.Client.Get(context.TODO(), bindingKey, existBinding)
 	if err == nil {
+		// If the spec.Placement is nil, this means that existBinding is generated by the dependency mechanism.
+		// If the spec.Placement is not nil, then it must be generated by PropagationPolicy.
+		if existBinding.Spec.Placement == nil {
```

Comment:
Increase this `IF` statement to determine the `existBinding`, make sure it was created by the `dependencies_distributor`,then we can modify `ConflictResolution`.